lib/apiservers/engine/backends/archive_test.go

```diff
@@ -99,34 +105,137 @@ func TestFindArchiveReaders(t *testing.T) {
 		{
 			containerSourcePath: "/",
 			expectedPrefices:    []string{"/", "/mnt/A", "/mnt/B", "/mnt/A/AB"},
+			expectedFilterSpecs: map[string]ReaderFilters{
+				"/": {
+					rebase:  "",
+					exclude: []string{"/mnt/A", "/mnt/B", "/mnt/A/AB"},
```

Comment:
I think we could only walk one level of subtree for the excludes as "/mnt/A/AB" should be redundant - no need for a change now though, and this is probably more resilient anyway.